exercises/perfect-numbers/canonical-data.json

```diff
@@ -120,24 +120,24 @@
             "cases": [
                 {
                     "uuid": "72445cee-660c-4d75-8506-6c40089dc302",
-                    "description": "Zero is rejected (not a natural number)",
+                    "description": "Zero is rejected (as it is not a positive integer)",
```

Comment:
Here I thought about adding "as it is" or similar.

Otherwise it's possible to misread it as "zero is rejected (instead of a natural number)"